### DISCUSSION TITLE
is discord.py back?

### DISCUSSION THREAD

**Discussion by oolio151** [Rapptz/discord.py#7581] [General] [answered]
I was looking at the repo, and there are many recent changes under a new user. Is this just fixes, or is discord.py under new ownership and being updated again? If it is coming back that's pretty exciting.

**Comment by izxxr** [comment]
[Yes.](https://gist.github.com/Rapptz/c4324f17a80c94776832430007ad40e6)

**Comment by LeoCx1000** [comment, marked as answer]
Yep it's back! No it hasn't changed owner. See [this gist](https://gist.github.com/Rapptz/c4324f17a80c94776832430007ad40e6) for Danny/Rapptz's reason on why he revived it.

Yes slash commands are supported.

**Reply by oolio151** [reply]
Thanks, this is pretty good news!

**Comment by Captain8771** [comment]
its back, danny is back to maintaining it, see [this gist](https://gist.github.com/Rapptz/c4324f17a80c94776832430007ad40e6) for details
